test/src/test/java/org/corfudb/integration/LogReplicationLogicalGroupIT.java

```diff
@@ -0,0 +1,728 @@
+        openLogReplicationStatusTable();
+    }
+
+    @Test
+    public void testBasicLogicalGroupReplication() throws Exception {
+        // Register client and add group destinations
+        CorfuRuntime clientRuntime = getClientRuntime();
+        LogReplicationLogicalGroupClient logicalGroupClient =
+                new LogReplicationLogicalGroupClient(clientRuntime, SAMPLE_CLIENT_NAME);
+        logicalGroupClient.addDestination(GROUP_A, DefaultClusterConfig.getSinkClusterIds().get(SINK2_INDEX));
+        logicalGroupClient.addDestination(GROUP_B, DefaultClusterConfig.getSinkClusterIds().get(SINK3_INDEX));
+
+        // Open tables for replication on Source side
+        int numTables = 2;
+        openFederatedTable(numTables, sourceCorfuStores.get(SOURCE_INDEX), srcFederatedTables);
+        openGroupATable(numTables, sourceCorfuStores.get(SOURCE_INDEX), srcTablesGroupA);
+        openGroupBTable(numTables, sourceCorfuStores.get(SOURCE_INDEX), srcTablesGroupB);
+        openFederatedTable(numTables, sinkCorfuStores.get(SINK1_INDEX), sinkFederatedTables);
+        openGroupATable(numTables, sinkCorfuStores.get(SINK2_INDEX), sinkTablesGroupA);
+        openGroupBTable(numTables, sinkCorfuStores.get(SINK3_INDEX), sinkTablesGroupB);
+
+        // Write data to Source side tables
+        writeDataOnSource(0, NUM_WRITES);
+
+        // Start log replication for all sessions
+        startReplicationServers();
+
+        // Verify all the sessions' snapshot sync completed
+        verifyInLogEntrySyncState(SINK1_INDEX, SessionManager.getDefaultSubscriber());
+        // TODO: should be replaced by real client name after Sink session creation workflow is introduced
+        verifyInLogEntrySyncState(SINK2_INDEX, SessionManager.getDefaultLogicalGroupSubscriber());
+        verifyInLogEntrySyncState(SINK3_INDEX, SessionManager.getDefaultLogicalGroupSubscriber());
+
+        // Verify tables' content on Sink side
+        verifyFederatedTableDataOnSink(sinkCorfuStores.get(SINK1_INDEX), NUM_WRITES, sinkFederatedTables);
+        verifyGroupATableDataOnSink(sinkCorfuStores.get(SINK2_INDEX), NUM_WRITES, sinkTablesGroupA);
+        verifyGroupBTableDataOnSink(sinkCorfuStores.get(SINK3_INDEX), NUM_WRITES, sinkTablesGroupB);
+
+        // Write more data for log entry sync
+        writeDataOnSource(NUM_WRITES, NUM_WRITES);
+
+        // Verify tables' content on Sink side
+        int targetWrites = 2 * NUM_WRITES;
+        verifyFederatedTableDataOnSink(sinkCorfuStores.get(SINK1_INDEX), targetWrites, sinkFederatedTables);
+        verifyGroupATableDataOnSink(sinkCorfuStores.get(SINK2_INDEX), targetWrites, sinkTablesGroupA);
+        verifyGroupBTableDataOnSink(sinkCorfuStores.get(SINK3_INDEX), targetWrites, sinkTablesGroupB);
+    }
+
+
+    /**
+     * This test has the similar workflow as testNewStreamsInLogEntrySync() in
+     * {@link LogReplicationDynamicStreamIT}, with Logical Group feature.
+     *
```

Comment:
If you prefer using link annotation pattern you could link to that method directly, otherwise the classname is linked while the method is left in plain text is a bit weird.

also, consider rename it to "LOGICAL_GROUP replication model" or session as we used commonly in other places, as didn't refer it as feature before